pubsub/google/cloud/pubsub_v1/publisher/batch/base.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
I would really love to see a class-level docstring here to describe exactly how the batching behavior works. It was very unclear to me when I first looked at this, and it's still unclear now.